src/main/java/com/autotune/analyzer/utils/AnalyzerErrorConstants.java

```diff
@@ -85,6 +85,7 @@ public static final class AutotuneObjectErrors {
         public static final String WRONG_TIMESTAMP = "The Start time should precede the End time!";
 
         public static final String MEASUREMENT_DURATION_ERROR = "Interval duration cannot be less than or greater than measurement_duration by more than " + KruizeConstants.TimeConv.MEASUREMENT_DURATION_THRESHOLD_SECONDS + " seconds";
+        public static final String MISSING_METRICS = "Metric data is not present for container : %s for experiment: %s";
```

Comment:
@khansaad - We do not have tests where complete metrics section is missing at present, these need to be added and validated against the expected response message.

Reply:
Also run the existing negative test that pass null for these metrics and check if the expected messages manually.